internal/libdrop_event_to_string.go

```diff
@@ -0,0 +1,35 @@
+package internal
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	norddrop "github.com/NordSecurity/libdrop-go/v7"
+	_ "github.com/NordSecurity/nordvpn-linux/fileshare/libdrop/symbols" // this is required to make cgo symbols available during linking
```

Comment:
Will cause problems when compiling without `libdrop.so` available. As this is specific to libdrop, whole code could be migrated under `libdrop` package

Reply:
Done :+1: , it's now in `event_manager.go`. I had it under `libdrop` package initially, but this introduces circular dependency and this helper is not used anywhere else than `event_manager.go`

Reply:
Done :+1: , this file `libdrop_event_to_string.go` no longer exists. I moved the helper method to `event_manager.go` as it's needed only there and could not put it in `libdrop` package because there was a circular dependency cycle:
```
package github.com/NordSecurity/nordvpn-linux/cmd/fileshare
        imports github.com/NordSecurity/nordvpn-linux/fileshare
        imports github.com/NordSecurity/nordvpn-linux/fileshare/libdrop
        imports github.com/NordSecurity/nordvpn-linux/fileshare: import cycle not allowed
```
but I moved the `EventToString` helper from libdrop package to fileshare package, so the part 
```
imports github.com/NordSecurity/nordvpn-linux/fileshare
        imports github.com/NordSecurity/nordvpn-linux/fileshare/libdrop
```
is no longer valid.